yarn.lock

```diff
@@ -1517,16 +1517,6 @@
     "@emotion/utils" "0.11.2"
     csstype "^2.5.7"
 
```

Comment:
Yarn lock changed without a package.json change. Is that intentional?

Reply:
Yes, intentional (kinda). This came out a merge conflict and now it's passing CI. Don't want to change although I can try adding it back.